demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
```

Comment:
I think its fine if you remove `/index` here.

Reply:
yes. Will remove.